herd/AArch64Sem.ml

```diff
@@ -563,7 +566,7 @@ module Make
           (fun _ -> mk_fault (Some a) dir an ii ft (Some "EL0"))  ii >>! B.Exit
 
       let an_xpte =
```

Comment:
Maybe `an_xpte` and `an_pte` could go into `AArch64Annot.ml`?

Reply:
Would those function also be called AArch64ASLSem.ml?

Reply:
Thinking twice, I'd rather keep those functions here, close to their calling sites.

Reply:
All right